app/src/README.md

```diff
@@ -1,17 +1,29 @@
-Node/Grunt modules for backend’s frontend workflow.
+# Bolts backend’s frontend workflow using grunt
```

Comment:
Yeah, totally understand. Please see my comment as doing just that not nitpicking. I really hate having to correct non-native speakers English.

A native speaker on the other hand… them I nitpick :grin: